resources/sample_files/Social_Test_1_Sample.csv

```diff
@@ -1,3 +1,4 @@
-Prem1,Prem2,Prem3,Prem4,District,PostTown,Postcode,CountryCode,Reference
-ADDRESS LINE 1,ADDRESS LINE 2,ADDRESS LINE 3,ADDRESS LINE 4,DISTRICT,POST TOWN,POSTCODE,E,LMS100001
-Office for National Statistics,Cardiff Road,Duffryn,Wales,Gwent District,Newport,NP10 8XG,E,LMS100002
\ No newline at end of file
+TLA,REFERENCE,COUNTRY,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,LOCALITY,TOWN_NAME,POSTCODE,UPRN

```

Comment:
This file is called `Social_Test_1_Sample.csv`. Perhaps it would be better to create a `Social_Pilot_Sample.csv` file and use that instead?